### ISSUE TITLE
关于defineModel的建议

### ISSUE THREAD

**Issue by xiankaiqun** [sxzz/unplugin-vue-macros#52]
### Clear and concise description of the problem

当前defineModel需要依赖新的实验性提案，该提案是否能通过还是未知数。
我认为是否可以通过一个writtebleComputed进行中介是否更加符合？

### Suggested solution

```vue
//====使用====
<script setup lang="ts">
// modelValue:WrittebleComputedRef<string>
const { modelValue } = defineModel<{
  modelValue: string
}>()

console.log(modelValue.value)
modelValue.value = 'newValue'


</script>

//====输出====
<script setup lang="ts">
const props = defineProps<{
  modelValue: string
}>();
    
const emit = defineEmits<{
  (event: 'update:modelValue', value: string): void
}>()

//中介
const modelValue1=computed({
    get:()=>props.modelValue,
    set:(value)=>emit('update:modelValue', value),
 })

//modelValue => modelValue1
console.log(modelValue1.value)
modelValue1.value = 'newValue'

</script>

```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/sxzz/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/sxzz/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by sxzz** [comment]
Reactivity Transform will be stable in Q3 2022. If Vue officially has a better solution, the difference shouldn't be that big.

Reactivity Transform 目前大概率是会在 Q3 2022 通过的。如果 Vue 官方有更好的方案，差别应该也不会很大。

**Comment by sxzz** [comment]
https://github.com/vueuse/vueuse/issues/2317#issuecomment-1276637755

TLDR: I will develop a version without Reactivity Transform later. It depends on VueUse.

**Comment by sxzz** [comment]
Done in #104
